doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,520 @@
+
+Once you have a URDF file, you can open the MoveIt Setup Assistant and import your URDF.
+This tutorial will guide you through a series of steps to configure various aspects of your robot,
+such as defining its kinematic structure, specifying planning groups and end-effectors, and collision checking related setting.
+To learn more about the URDF and SRDF, you can refer to the `URDF SRDF Overview <https://moveit.picknik.ai/humble/doc/examples/urdf_srdf/urdf_srdf_tutorial.html#>`_ page.
 
 Getting Started
-------------------------
+---------------
 
-MoveIt and ROS
+MoveIt and ROS 2
 
-* Follow the instructions for :moveit_website:`installing MoveIt<install>`
+* Follow the `instructions for installing MoveIt <https://moveit.picknik.ai/humble/doc/tutorials/getting_started/getting_started.html>`_
   first if you have not already done that.
 
-* If you haven't already done so, make sure you have the `Franka description
-  package <https://github.com/frankaemika/franka_ros>`_ for Noetic: ::
-
-    sudo apt install ros-noetic-franka-description
-
-* If you have the ``panda_moveit_config`` package already git-cloned from the *Getting Started* page, be sure to delete that now since this tutorial will teach you how to create it from scratch: ::
-
-   cd ~/ws_moveit/src
-   rm -rf panda_moveit_config
-   catkin clean panda_moveit_config
+* We use the `moveit_resources_panda_description <https://github.com/ros-planning/moveit_resources/tree/humble/panda_description/urdf>`_
+  package for Humble. This package should already be included in your workspace if you've completed the MoveIt installation instructions.
 
 Step 1: Start
----------------
+-------------
 
 * To start the MoveIt Setup Assistant: ::
 
-   roslaunch moveit_setup_assistant setup_assistant.launch
+   ros2 launch moveit_setup_assistant setup_assistant.launch.py
 
-* This will bring up the start screen with two choices: *Create New
-  MoveIt Configuration Package* or *Edit Existing MoveIt
-  Configuration Package*.
+* This will bring up the start screen with two choices: **Create New
+  MoveIt Configuration Package** or **Edit Existing MoveIt
+  Configuration Package**.
 
-* Click on the *Create New MoveIt Configuration Package* button to
+* Click on the **Create New MoveIt Configuration Package** button to
   bring up the following screen:
 
-.. image:: setup_assistant_start.png
-
-* Click on the browse button and navigate to the *panda_arm_hand.urdf.xacro* file
-  installed when you installed the Franka package above. (This file
-  gets installed in
-  /opt/ros/noetic/share/franka_description/robots/panda_arm_hand.urdf.xacro on Ubuntu
-  with ROS Noetic.)  Choose that file and then click *Load Files*. The
+.. image:: setup_assistant_create_package.png
+   :width: 700px
+   :align: center
+
+* Click on the **Browse** button and navigate to the *panda.urdf* file
+  from the *moveit_resources_panda_description package* available in the following path: :: 
```

Comment:
For code, use double backticks to it's formatted as `panda.urdf` instead of _panda.urdf_